src/components/form-elements/_form-elements.scss

```diff
@@ -295,7 +294,7 @@ $textareaScaleFactor: 2/3;
   }
   &--small {
     height: 36px;
-    padding: $textareaPadding*$textareaScaleFactor $textareaPadding;
```

Comment:
old value was 7.3333333px which was a bit too much to accomoddate everything and caused scrollbar to appear